README.md

```
@@ -193,27 +279,31 @@
+  <img height="400" alt="PWA" src="https://github.com/user-attachments/assets/c6564e26-dd98-4bf3-aa14-63421c2fc1d8" />
+  <img height="400" alt="PWA" src="https://github.com/user-attachments/assets/59e281d8-fa10-421c-901e-35ca52f56458" />
+</p>
 
 <br/>
 
 ## 기술 스택
 
-### 프레임워크 & 스타일
+<h3>프레임워크 & 스타일</h3>
 <p>
   <img src="https://img.shields.io/badge/React-61DAFB?style=for-the-badge&logo=react&logoColor=white"/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)